webapp/src/containers/Rhs/projectDetails/header.tsx

```diff
@@ -88,6 +114,21 @@ const Header = ({projectDetails, showAllSubscriptions, handlePagination, setShow
         }
     }, [filter.serviceType]);
 
+    const handleCheckboxChange = (e: ChangeEvent<HTMLInputElement>) => {
+        setDeleteSubscriptions(e.target.checked);
+    };
+
+    const deleteSubscriptionsCheckbox = (
+        <>
+            <input
+                type='checkbox'
+                id='deleteSubscriptions'
+                onChange={handleCheckboxChange}
+            />
+            <label>{pluginConstants.common.deleteAllSubscriptionsMessage}</label>
```

Comment:
wrap these into a `div` with `flex` and vertically align them center, also put a gap of `5px` between checkbox and lebel